models/en-CA.json

```diff
@@ -1,7 +1,7 @@
 {
     "interactionModel": {
         "languageModel": {
-            "invocationName": "rocket chat",
+            "invocationName": "chat rocket",
```

Comment:
I think 'Rocket Chat' is more appropriate than 'Chat Rocket'. 🤔

Reply:
Thanks for pointing. Missed to update this.